### PR TITLE
Use hdmf common schema 1.5.0 and update release notes

### PR DESCRIPTION
**Pull request by rly**
Use hdmf common schema 1.5.0 and update release notes (tentative release April 23)

### CODE REVIEW

**Comment by rly** [comment]
We had discussed removing the hdmf-common-schema git submodule from this repo because this version of hdmf-common is not loaded by PyNWB, but rather, the version cached in the local copy of HDMF is loaded. So having this copy of hdmf-common may be confusing. However until the schema language supports a 'version' key: https://github.com/hdmf-dev/hdmf-schema-language/issues/11 , I think it is useful to include in the nwb schema a copy of a known compatible version of HDMF common.

File: docs/format/source/format_release_notes.rst [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
-  - Fix documentation for ragged arrays.
+- Add optional "continuity" field to ``TimeSeries``.
+- Update hdmf-common-schema from 1.1.3 to version 1.5.0.
+  - The HDMF-experimental namespace was added, which includes the ``ExternalResources`` and ``EnumData``
+    data types. Schema in the HDMF-experimental namespace are experimental and subject to breaking changes at any time.
+  - Added experimental data type ``ExternalResources`` for storing ontology information / external resource references.
+  - Added experimental data type ``EnumData`` to store data from a set of fixed values.
+  - Changed dtype for datasets within ``CSRMatrix`` from 'int' to 'uint' and added missing `data_type_inc: Container`
+    to the ``CSRMatrix`` type.
+    Added data type ``SimpleMultiContainer``, a Container for storing other Container and Data objects together.
```
Comment (oruebel) [inline docs/format/source/format_release_notes.rst]:
```suggestion
    - Added data type ``SimpleMultiContainer``, a Container for storing other Container and Data objects together.
```

File: docs/format/source/format_release_notes.rst [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
+  - Fixed missing dtype for ``VectorIndex``.
```
Comment (oruebel) [inline docs/format/source/format_release_notes.rst:22]:
```suggestion
  - Added data type ``AlignedDynamicTable``, a DynamicTable type with support for categories (or sub-headings) each described by a separate DynamicTable.
  - Fixed missing dtype for ``VectorIndex``.
```

**Review by oruebel — DISMISSED** [review]

File: docs/format/source/format_release_notes.rst (hunk 2 above)
Comment (rly) [reply docs/format/source/format_release_notes.rst:22]: Oops I forgot this as I was making readthedocs for HDMF common 1.5.0 work. Thanks!

File: docs/format/source/format_release_notes.rst [hunk 3]
```diff
@@ -1,23 +1,26 @@
...
-  - Fix documentation for ragged arrays.
+- Add optional "continuity" field to ``TimeSeries``.
+- Update hdmf-common-schema from 1.1.3 to version 1.5.0.
+  - The HDMF-experimental namespace was added, which includes the ``ExternalResources`` and ``EnumData``
+    data types. Schema in the HDMF-experimental namespace are experimental and subject to breaking changes at any time.
+  - Added experimental data type ``ExternalResources`` for storing ontology information / external resource references.
+  - Added experimental data type ``EnumData`` to store data from a set of fixed values.
+  - Changed dtype for datasets within ``CSRMatrix`` from 'int' to 'uint' and added missing `data_type_inc: Container`
+    to the ``CSRMatrix`` type.
+    - Added data type ``SimpleMultiContainer``, a Container for storing other Container and Data objects together.
```
Comment (rly) [inline docs/format/source/format_release_notes.rst]:
```suggestion
  - Added data type ``SimpleMultiContainer``, a Container for storing other Container and Data objects together.
```

**Review by oruebel — DISMISSED** [review]

**Review by oruebel — APPROVED** [review]

**Pull request merged**